internal/usecase/path_sign_txn.go

```diff
@@ -151,10 +155,17 @@ func (b *Backend) signTx(
 		return nil, err
 	}
 
+	v, r, s := signedTx.RawSignatureValues()
+
 	return &logical.Response{
 		Data: map[string]interface{}{
 			"txHash":   signedTx.Hash().Hex(),
 			"signedTx": hexutil.Encode(signedTxBuff.Bytes()),
+			"signature": map[string]interface{}{
```

Comment:
why are r,v,s being explicitly passed when they are already part of signedTxBuff ?